docs/how_to/perf.md

```diff
@@ -1,7 +1,155 @@
 # Performance
 
-Here are some tips to get the most out of MXNet:
-## Data
+Performance is mainly affected by the following 4 factors:
+
+1. Implementation of operators (Convolution, Pooling, ..)
+   - [Intel CPU](#intel-cpu)
+   - [Nvidia GPU](#nvidia-gpu)
+2. Input data loading and augmentation
+   - [Input Data](#input-data)
+3. Workloads (computation graph) optimization and scheduling
+   - [Profiler](#profiler)
+4. Communication for multi-devices training
+   - [Multiple Devices](#multiple-devices)
+
+## Intel CPU
+
+For using Intel Xeon CPUs for training and inference, we suggest to enable
+both `USE_MKL2017 = 1` and `USE_MKL2017_EXPERIMENTAL = 1` in
+`config.mk`. Check
+[MKL_README.md](https://github.com/dmlc/mxnet/blob/master/MKL_README.md) for
+details
+
+Also setting the following two environment variables may help:
+- `KMP_AFFINITY=granularity=fine,compact,1,0` if there are two physical CPUs
+- `OMP_NUM_THREADS=vCPUs / 2` in which `vGPUs` is the number of virtual CPUs.
+  For linux we can get it by `cat /proc/cpuinfo  | grep processor | wc -l`
+
+Note that MXNet treats all GPU in a single machine as a single device. So when
+specify `gpu(0)` or `gpu()`, all GPU cores in the machine will be used.
+
+### Scoring results
+The following table shows the scoring performance, namely number of images can
+be predicted per second. We used AWS EC2 C4.8xlarge (dual Intel(R) Xeon(R) CPU
+E5-2666 v3 @ 2.90GHz) and
+[example/image-classification/benchmark_score.py](https://github.com/dmlc/mxnet/blob/master/example/image-classification/benchmark_score.py)
+with MXNet commit `0a03417`
+
+| Batch | Alexnet | VGG | Inception-BN | Inception-v3 | Resnet 50 | Resnet 152 |
+| --- | --- | --- | --- | --- | --- | --- |
+|   1 |  98.44 | 24.66 |  27.43 |  11.91 |  38.74 | 14.95 |
+|   2 | 121.67 | 27.07 |  24.78 |  11.61 |  44.27 | 15.62 |
+|   4 | 254.39 | 38.51 |  28.76 |  12.86 |  40.88 | 19.01 |
+|   8 | 237.73 | 36.97 |  25.57 |  12.68 |  43.00 | 16.11 |
+|  16 | 280.82 | 40.00 |  20.85 |  11.77 |  55.00 | 16.93 |
+|  32 | 285.41 | 44.40 |  31.03 |  12.45 |  55.70 | 17.02 |
+
+## Nvidia GPU
+
+`CUDNN` often greatly accelerate performance on Nvidia GPUs, check a recent
```

Comment:
I would suggest to mentioned something like ' cuDNN often greatly accelerate performance especially Convolution layer's performance.'